### ISSUE TITLE
@statoscope/webpack-plugin does not work without statsOptions.context

### ISSUE THREAD

**Issue by ValeraS** [statoscope/statoscope#113]
**Describe the bug**
if not specify statsOptions.context, webpack build fails with error:
> TypeError: Cannot read property 'context' of undefined 

**Expected behavior**
no error

**Versions**:
 - Webpack: 4
 - Statoscope: 5.9.0

@statoscope/webpack-plugin v5.8.0 works without error.

**Comment by smelukov** [comment]
@ValeraS can't reproduce :(
Can you provide Statosope plugin config example?

**Comment by smelukov** [comment]
@ValeraS fixes in 5.9.1
Thanks for the issue 🙏

**Comment by KubaZ** [comment]
@smelukov issue was fixed in @statoscope/webpack-plugin@5.9.1, but from what i investigated, it should be fixed in @statoscope/webpack-stats-extension-package-info package. Because of that previous versions of @statoscope/webpack-plugin still doesn't work, for example issue still exists in @statoscope/webpack-plugin@5.6.2
```
TypeError: Cannot read property 'context' of undefined
    at WebpackCompressedExtension.handleCompiler (/node_modules/@statoscope/webpack-stats-extension-package-info/dist/index.js:22:100)
    at StatoscopeWebpackPlugin.apply (/node_modules/@statoscope/webpack-plugin/dist/index.js:42:30)
```

**Comment by smelukov** [comment]
@KubaZ thank 🙏  Fixed in 5.10.2

**Comment by KubaZ** [comment]
Thanks @smelukov for lighting fast fix, `@statoscope/webpack-plugin@5.6.2` is working again ;)

**Comment by smelukov** [comment]
@KubaZ btw, you could update all of statoscope packages that you use to latest versions 😁
There are some cool features (e.g. new stats validators and ability to pass any custom reports to the UI)
